### DISCUSSION TITLE
Frequent Parkside charger voltage updates

### DISCUSSION THREAD

**Discussion by mrsofa** [make-all/tuya-local#2199] [Q&A]
Hi, a simple question, 
I have the Parkside PLGS2012A1 charger integrated via tuya local in to my HA, I have problem that the entity 

number.parkside_plgs2012a1_charger_charge_voltage

changes it's state too often - every 3 seconds, that results in a mess in the logbook, making it difficult to debug my other problems. 

How can I achieve that it doesn't update so often? 
![image](https://github.com/user-attachments/assets/c293cce1-9824-49fb-869c-653d9253c002)

**Comment by make-all** [comment]
You need to configure [Recorder](https://www.home-assistant.io/integrations/recorder/) to ignore the entity if you don't want to see such messages in your logbook.
